doc/source/operating.rst

```diff
@@ -36,6 +37,10 @@ The following uwsgi configuration file can be used::
   plugins = python
   buffer-size = 65535
   lazy-apps = true
+  add-header = Connection: close
+
+You should configure the number of process and threads according to the number
+of CPU you have, usually around 1.5 × number of CPU.
```

Comment:
isn't really clear. should the processes be 1.5x or the threads? or both? for me, the threads didn't really do anything